pavelib/bok_choy.py

```diff
@@ -19,6 +19,7 @@
 BOKCHOY_OPTS = [
     ('test_spec=', 't', 'Specific test to run'),
     ('fasttest', 'a', 'Skip some setup'),
+    ('skip_clean', 'C', 'Skip cleaning repository before running tests'),
```

Comment:
It doesn't need to be, but I used the capital C to keep it consistent with the usage in other paver commands. This arg is available in the `paver test_js` already and it uses a capital `-C` for `--skip_clean` and lower case `-c` for `--coverage`.